app/controllers/revision_feedback_controller.rb

```diff
@@ -1,16 +1,16 @@
 # frozen_string_literal: true
 
 require_dependency "#{Rails.root}/lib/revision_feedback_service"
-require_dependency "#{Rails.root}/lib/importers/revision_score_importer"
+require_dependency "#{Rails.root}/lib/lift_wing_api"
 
 class RevisionFeedbackController < ApplicationController
   def index
     set_latest_revision_id
     return if @rev_id.nil?
-    revision_data = RevisionScoreImporter.new.fetch_data_for_revision_id(@rev_id)
-    @feedback = RevisionFeedbackService.new(revision_data[:features]).feedback
+    revision_data = LiftWingApi.new(@wiki).get_revision_data([@rev_id])[@rev_id.to_s]
+    @feedback = RevisionFeedbackService.new(revision_data['features']).feedback
     @user_feedback = Assignment.find(params['assignment_id']).assignment_suggestions
-    @rating = revision_data[:rating]
+    @rating = revision_data['prediction']
```

Comment:
I think this is the easiest way to fix that for now. Instead of using the `RevisionScoreImporter`, it uses the `LiftWingApi` directly. This is because the `RevisionScoreImporter` won't return the LiftWing features or the predictions for `en.wikipedia` (and this controller ony works for `en.wikipedia`).

I can still create an issue to move this to the frontend if we think that's better.